config/typescript/modern.json

```diff
@@ -19,7 +19,8 @@
 			"DOM",
 			"webWorker"
 		],
-		"moduleResolution": "node",
+		"module": "node16",
```

Comment:
nice

Reply:
This was **required** for the usage of jest modules importing:

```js
// example.test.js
const Example = require('@{scope}/{name}'); // works now and does not throw an error in tests
const Example = require('../../dist/module/index'); // no longer required to do this
```